language/tools/move-cli/tests/cli_testsuite.rs

```diff
@@ -7,9 +7,21 @@ use std::path::Path;
 
 pub const CLI_BINARY: &str = "../../../target/debug/move-cli";
 pub const CLI_TESTSUITE_DIR: &str = "tests/testsuite";
+pub const CLI_TESTSUITE_SAMPLE: &str = "events_emit_view";
 
-fn run_all(args_path: &Path) -> datatest_stable::Result<()> {
-    Ok(test::run_one(args_path, CLI_BINARY)?)
+#[test]
+fn test_run_all() {
+    assert!(test::run_all(Path::new(CLI_TESTSUITE_DIR), CLI_BINARY).is_ok());
 }
 
-datatest_stable::harness!(run_all, CLI_TESTSUITE_DIR, r"args.txt$");
+#[test]
+fn test_run_one_dir() {
+    let path_dir = format!("{}/{}", CLI_TESTSUITE_DIR, CLI_TESTSUITE_SAMPLE);
```

Comment:
Nit: using `p = Path::new(CLI_TESTSUITE_DIR); p.join(CLI_TESTSUITE_SAMPLE)` instead will ensure that this works on Windows etc.

Reply:
Shall I change the path for CLI_BINARY and CLI_TESTSUITE_DIR too?

Reply:
I think we should eventually make a `move_lang::file_utils` or maybe even a separate crate

I have a `path_buf_dir` which makes doing these things a lot easier

Reply:
(Don't need `path_dir` macro for this PR was more a general comment)